src/main/java/me/imoltres/bbu/game/threads/GameThread.kt

```diff
@@ -57,7 +73,57 @@ class GameThread(val game: Game) : Thread() {
                             shrinkBorder(getBorderShrinkTime(border))
                             shrinking = !shrinking
                         }
+                        //check and see if any teams beacons are out of the border
+                        for (team in game.getTeams(true)) {
```

Comment:
Check this in a listener rather than in the GameThread

Reply:
This cannot be changed to a listener because there is no event for border shrink
I will however make this run every 5 seconds instead of every second

Reply:
my problem was that it was running every 50ms, not every second

Reply:
Alright yeah I see that.